Gems/EMotionFX/Code/EMotionFX/Source/MotionData/RootMotionExtractionData.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Memory/SystemAllocator.h>
+
+namespace EMotionFX
+{
+    class RootMotionExtractionData
```

Comment:
suggestion: An API comment here would be super useful

Reply:
added